### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Gyshie**
Updated the readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,7 @@
 # IBAN Examples
+
+This repository consists of examples of IBANs.
+
+Feel free to contribute by adding Valid IBAN examples to the file for the appropriate country. 🤍
+
+IBAN stands for International Bank Account Number and is an internationally accepted numbering system that identifies individual bank accounts worldwide. Banks use IBANs to process money transfers and ensure payments arrive in the correct location. You may need to use an IBAN to make a bank transfer within Spain or internationally.
```
Comment (Vulwsztyn) [inline README.md:7]: Remove this please

Comment (Vulwsztyn) [reply README.md:7]: . (comment needed to request changes by Github, sry)

**Review by Vulwsztyn — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Vulwsztyn) [reply README.md:7]: ping

**Comment by Gyshie** [comment]
Hello @Vulwsztyn you want me to remove my suggestion or what? I didn't get you. Can you please classify it?

**Comment by Vulwsztyn** [comment]
> Hello @Vulwsztyn you want me to remove my suggestion or what? I didn't get you. Can you please classify it?

Yeah, sorry. I want you to remove last line of text from readme. It is not the aim of this repo to tell people what IBAN is. I'm sorry for inconvenience and thankful for your contribution.

**Comment by Gyshie** [comment]
# IBAN Examples

This repository consists of examples of IBANs.

Feel free to contribute by adding Valid IBAN examples to the file for the appropriate country. 🤍

**Comment by Gyshie** [comment]
Like above?

**Comment by Vulwsztyn** [comment]
> Like above?

Yes.

Why did you close this PR? You could have added a new commit removing the line.

Are you going to commit the IBAN examples you added in this PR in a new PR?

**Comment by Gyshie** [comment]
Hope you got the new PR :)

**Comment by Vulwsztyn** [comment]
> Hope you got the new PR :)
Only the one with readme changes not the ones with cw and ky IBANs

**Comment by Gyshie** [comment]
Oh! I'll resend em. Sry

**Comment by Gyshie** [comment]
I made it as a new PR.

**Pull request closed without merging**